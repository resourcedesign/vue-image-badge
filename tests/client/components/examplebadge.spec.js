const Vue = require('vue')
const utils = require('../utils')

const ExampleBadge = require('../../../src/VueBadge.vue')

describe('Example Badge', () => {
  it('should mount successfully', () => {
    const vm = new Vue({
      el: document.createElement('div'),
      render: (h) => h(ExampleBadge)
    })
    const badgeInstance = vm.$children[0]
    expect(badgeInstance._isMounted).toBe(true)

    return utils.expectToMatchSnapshot(vm)
  })
})
