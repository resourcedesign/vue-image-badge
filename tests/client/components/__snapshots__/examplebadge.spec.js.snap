exports[`Example Badge should mount successfully 1`] = `
"img(src=\'\')
"
`;
